openghg/store/base/_base.py

```diff
@@ -160,6 +160,24 @@ def check_hashes(self, filepaths: multiPathType, force: bool) -> Tuple[Dict[str,
 
         return seen, unseen
 
+    def add_metadata_keys(self, data: Dict, additional_metadata: dict) -> dict:
```

Comment:
You're right we should be consistent - I included `Dict` for this as that was what the `data` input to `assign_data` was so I'm actually going to update it to:
```suggestion
    def add_metadata_keys(self, data: Dict, additional_metadata: Dict) -> Dict:
```

Reply:
Since python 3.9 "Dict" etc has been deprecated: https://peps.python.org/pep-0585/

I guess it doesn't really matter because at some point we'll probably want to use a code formatter to update all the type hints (e.g. https://pypi.org/project/pyupgrade/).

Reply:
Ok - agreed this should be updated then in the longer term. For now I'll keep it as is to be consistent with the other uses of `data` but yes this should all get updated in one go really to replace `Dict` with `dict` where this is used.